src/main/java/com/example/techiteasyspringhomework/models/Product.java

```diff
@@ -0,0 +1,56 @@
+package com.example.techiteasyspringhomework.models;
+
+public abstract class Product {
+
+    // make id int generated as identity?
```

Comment:
Ja klopt, Er zou alleen nog een annotatie boven product moeten (omdat het wel data is wat we willen opslaan, namelijk de @MappedSuperclass)